package.json

```diff
@@ -46,7 +46,7 @@
     "copy-webpack-plugin": "4.0.1",
     "coveralls": "^2.11.15",
     "css-loader": "^0.26.0",
-    "d3": "^4.4.1",
+    "d3": "^4.9.1",
```

Comment:
ui-classic forces `d3` to be `~3.5.0` (`3.5.5` right now) anyway ;)

Does this PR actually depend on d3 4+?